ansible/examples/slave_libvirt.yml

```diff
@@ -0,0 +1,235 @@
+---
+
+- hosts: all
+  sudo: true
+  user: ubuntu
+  vars:
+   - jenkins_user: 'jenkins-build'
+   # jenkins API credentials:
+   - api_user: 'ceph-jenkins'
+   - token: '{{ token }}'
+   - api_uri: 'https://jenkins.ceph.com'
+   - nodename: '{{ nodename }}'
+   - labels: '{{ labels }}'
+   - use_jnlp: false
+
+  tasks:
+
+    # vagrant doesn't have repositories, this chacra repo will be better to have
+    # around and can get updates as soon as a new vagrant version is published via
+    # chacractl
+    - name: add the vagrant repository
+      sudo: yes
+      apt_repository:
+        repo: "deb [trusted=yes] https://chacra.ceph.com/r/vagrant/latest/HEAD/ubuntu/xenial/flavors/default/ main"
```

Comment:
It's a bummer ``apt_repository`` doesn't take a url so you could use: https://chacra.ceph.com/repos/vagrant/latest/HEAD/ubuntu/xenial/repo/